project-fob/Controllers/HostController.cs

```diff
@@ -48,13 +44,13 @@ public ActionResult Finish(string message)
         public ActionResult Leave(string message)
         {
             var gotvalue = HttpContext.Session.TryGetValue("sessionid", out var session);
-            string byteArrayToString = System.Text.Encoding.ASCII.GetString(session);
+            string UserIdString = System.Text.Encoding.ASCII.GetString(session);
```

Comment:
👍 this is much nicer naming to say what things are. Aside: C# style is usual camelCase for variables (including params and fields), and TitleCase for methods and properties.